lib/al/Library/Collision/PartsConnector.h

```diff
@@ -0,0 +1,79 @@
+void setConnectorBaseQuatTrans(al::LiveActor*, al::MtxConnector*);
+void connectPoseQT(al::LiveActor*, const al::MtxConnector*);
+void connectPoseQT(al::LiveActor*, const al::MtxConnector*, const sead::Quatf&, const sead::Vector3f&);
+void connectPoseTrans(al::LiveActor*, const al::MtxConnector*, const sead::Vector3f&);
+void connectPoseMtx(al::LiveActor*, const al::MtxConnector*, const sead::Matrix34f&);
+void calcConnectTrans(sead::Vector3f*, const al::MtxConnector*, const sead::Vector3f&);
+void calcConnectTrans(sead::Vector3f*, const al::MtxConnector*);
+void calcConnectDir(sead::Vector3f*, const al::MtxConnector*, const sead::Vector3f&);
+void calcConnectQT(sead::Quatf*, sead::Vector3f*, const al::MtxConnector*, const sead::Quatf&, const sead::Vector3f&);
+void calcConnectQT(sead::Quatf*, sead::Vector3f*, const al::MtxConnector*);
+void calcConnectMtx(sead::Matrix34f*, const al::MtxConnector*, const sead::Matrix34f&);
+void calcConnectMtx(sead::Matrix34f*, const al::MtxConnector*, const sead::Quatf&, const sead::Vector3f&);
+void attachMtxConnectorToCollisionRT(al::MtxConnector*, const al::LiveActor*, bool, bool);
+void attachMtxConnectorToCollisionQT(al::MtxConnector*, const al::LiveActor*, bool, bool);
+void attachMtxConnectorToJoint(al::MtxConnector*, const al::LiveActor*, char const*);
+void attachMtxConnectorToJoint(al::MtxConnector*, const al::LiveActor*, char const*, const sead::Vector3f&, const sead::Vector3f&);
+void attachMtxConnectorToMtxPtr(al::MtxConnector*, const sead::Matrix34f*, const sead::Vector3f&, const sead::Vector3f&);
+void attachMtxConnectorToActor(al::MtxConnector*, const al::LiveActor*);
+void attachMtxConnectorToActor(al::MtxConnector*, const al::LiveActor*, const sead::Vector3f&, const sead::Vector3f&);
+void attachMtxConnectorToMtxPtr(al::MtxConnector*, const sead::Matrix34f*);
+void attachMtxConnectorToSensor(al::MtxConnector*, al::HitSensor*, const sead::Matrix34f&);
+void attachSensorConnectorToSensor(al::SensorConnector*, al::HitSensor*, const sead::Matrix34f&);
+void attachMtxConnectorToScreenPointTarget(al::MtxConnector*, al::ScreenPointTarget*, const sead::Matrix34f&);
+void attachCollisionPartsConnector(al::CollisionPartsConnector*, const al::CollisionParts*);
+void attachCollisionPartsConnectorToGround(al::CollisionPartsConnector*, const al::LiveActor*);
+void attachToHitTriangle(al::CollisionPartsConnector*, const al::Triangle&, const sead::Matrix34f&);
+void attachToHitInfo(al::CollisionPartsConnector*, const al::HitInfo&, const sead::Matrix34f&);
+void attachToHitInfoNrmToMinusZ(al::CollisionPartsConnector*, const al::HitInfo&);
+void calcConnectInfo(const al::MtxConnector*, sead::Vector3f*, sead::Quatf*, sead::Vector3f*, const sead::Vector3f&, const sead::Vector3f&);
+void connectPoseQTUsingConnectInfo(al::LiveActor*, const al::MtxConnector*);
+void getConnectBaseQuat(const al::MtxConnector*);
+void getConnectBaseTrans(const al::MtxConnector*);
```

Comment:
At least these two have a wrong return type, not sure about other functions in this class